docs/ref/schema-editor.txt

```diff
@@ -184,3 +184,9 @@ accessed.
 
 This is useful when doing data migrations for :ref:`migrations with multiple
 databases <data-migrations-and-multiple-databases>`.
+
+Template strings
+----------------
+
+The ``SchemaEditor`` contains a number of template strings for creating and
```

Comment:
There are quite a lot of those strings, `sql_create_index` being one. I could describe all of them, but I think that goes a bit beyond of the scope of this PR. I'll rework a bit; perhaps showing some examples is better.